### ISSUE TITLE
Broken links for bandit suggestions in codeguru results

### ISSUE THREAD

**Issue by virajvchaudhari** [aws-actions/codeguru-reviewer#18]
The bandit suggestions after codeguru analysis for api usage recommendations are display with broken links.

Example: 
`Consider possible security implications associated with subprocess module. [https://bandit.readthedocs.io/en/latest/blacklists/blacklist\_imports.html#b404-import-subprocess](https://bandit.readthedocs.io/en/latest/blacklists/blacklist%5C_imports.html#b404-import-subprocess)`

The underscores are not handled correctly during link formatting. It would be great if this could be resolved. 

So basically this should be: 
Broken: https://bandit.readthedocs.io/en/latest/blacklists/blacklist\_imports.html#b404-import-subprocess
Fix: https://bandit.readthedocs.io/en/latest/blacklists/blacklist_imports.html#b404-import-subprocess

**Comment by virajvchaudhari** [comment]
Hello, do we have any updates on this?
